File: daw.webapp12/src/main/java/com/daw/webapp12/entity/Blog.java
package com.daw.webapp12.entity;

import javax.persistence.*;
import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;

@Entity
@Table(name = "blog")
public class Blog {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;


    @NotNull
    private String title;

    private String text;

    public Blog(){}

    public Blog(String title, String text) {
        this.title = title;
        this.text = text;
    }

    public long getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getText() {
        return text;
    }

    public void setText(String email) {
        this.text = text;
    }
}

File: WebApp12Angular/src/app/entity/blog.ts


export class Blog{
    id?: number;
    title: string;
    text: string;
    images : File;


    constructor(title:string, text:string, images:File){
        this.title = title;
        this.text = text;
        this.images = images;
    }

    getId() {
        return this.id;
    }

    setId(id:number) {
        this.id = id;
    }

    getTitle() {
        return this.title;
    }

    setTitle(title:string) {
        this.title = title;
    }

    getText() {
        return this.text;
    }

    setText(text: string) {
        this.text = text;
    }

    getImages(){
        return this.images;
    }

    setImages(images:File){
        this.images = images;
    }
}